docs/architecture/self-hosted/installation-helm/installation-helm.mdx

```diff
@@ -16,24 +16,27 @@ Installation requirements and Helm chart values are outlined in this page.
 
 For an example step-by-step installation, visit [Installation on AWS](install-on-aws).
 
-## Requirements
+### Kubernetes cluster requirements
 
-To install Prophecy, you need:
+- **Kubernetes Version**: 1.21 or later.
+- **Namespace**: Dedicated namespace for Prophecy installation.
+- **Permissions**: Ability to install custom resources (CRDs) or deploy an additional provided Helm chart.
+- **Cluster Configuration**: Supports multi-AZ or single-AZ mode.
+- **Block Storage**: Required; in multi-AZ mode, set volume binding mode to `waitforfirstconsumer`.
+- **Persistent Storage**: Dynamic provisioning (e.g., gp2) required.
 
-- Kubernetes version 1.21 or later.
-- A namespace in Kubernetes cluster where you will install Prophecy.
-- Permission to install custom resources, or CRDs. Alternatively, you can deploy a single Helm chart that can be shared on request.
-- Block storage in the Kubernetes cluster.
-- Kubernetes cluster configured in either **multi-AZ** or **single-AZ** mode. For multi-AZ mode, the block storage has to have the volume binding mode set to `waitforfirstconsumer`.
-- Persistent storage with dynamic provisioning (like `gp2`).
-- [Helm](https://helm.sh/docs/intro/quickstart/) for Prophecy installation.
+### Helm requirements
 
-Minimum service requirements:
+- **[Helm](https://helm.sh/docs/intro/quickstart/)**: Must be installed to deploy Prophecy.
+- **Custom Domain Hosting**: Requires SSL certificates and DNS configuration; include during Helm installation.
```

Comment:
We can try something like this here:- 

- **[Custom Domain Hosting](#custom-domain-names)**: